src/components/calcite-tip/calcite-tip.tsx

```diff
@@ -4,6 +4,7 @@ import { getItem, setItem } from "../utils/localStorage";
 import CalciteIcon from "../utils/CalciteIcon";
 import { CalciteTheme } from "../interfaces";
 import { CSS, TEXT } from "./resources";
+import { VNode } from "@esri/calcite-components/dist/types/stencil.core";
```

Comment:
Nitpick: I think we should import this from `"@stencil/core/dist/declarations";`

Reply:
Updated for all files.